frontend/components/commons/dataset-description/DatasetDescription.component.vue

```diff
@@ -1,23 +1,69 @@
 <template>
   <div class="description">
     <h2 class="--heading5 --semibold description__title">{{ title }}</h2>
-    <p class="--body1 description__text">{{ datasetDescription }}</p>
+    <BaseSpinner v-if="isLoading" />
+    <markdown-editor
+      v-if="!isLoading"
+      :value="datasetGuidelines"
+      placeholder="Insert here your guidelines"
+      @save="updateGuidelines"
+    />
   </div>
 </template>
 
 <script>
+import { getDatasetFromORM } from "@/models/dataset.utilities";
+import MarkdownEditor from './MarkdownEditor.component'
+import { mapActions } from "vuex";
+
 export default {
   props: {
-    datasetDescription: {
+    datasetId: {
+      type: Array,
+      required: true,
+    },
+    datasetTask: {
       type: String,
       required: true,
     },
+    isLoading: {
+      type: Boolean,
+      default: () => true,
+    },
+  },
+  components: {
+    MarkdownEditor
   },
```

Comment:
in Nuxt, you are not obliged to import the component. So the `components` section can be remove